models/attachment.go

```diff
@@ -50,6 +66,20 @@ func (a *Attachment) LocalPath() string {
 	return AttachmentLocalPath(a.UUID)
 }
 
+// Size returns the file's size of the attachment
+func (a *Attachment) Size() (int64, error) {
+	fi, err := os.Stat(a.LocalPath())
+	if err != nil {
+		return 0, err
+	}
+	return fi.Size(), nil
+}
+
+// DownloadURL returns the download url of the attached file
+func (a *Attachment) DownloadURL() string {
+	return fmt.Sprintf("%sattachments/%s", setting.AppURL, a.UUID)
```

Comment:
Here

Reply:
This is unrelated since it is used to generade the download link and not for API usage.